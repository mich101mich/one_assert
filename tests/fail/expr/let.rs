fn main() {
    one_assert::assert!(let);
    one_assert::assert!(let b);
    one_assert::assert!(let b = true);
    one_assert::assert!(let b = 1 +);
}
