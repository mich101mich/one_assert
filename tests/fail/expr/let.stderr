error: incomplete expression: unexpected end of input, expected one of: identifier, `::`, `<`, `_`, literal, `const`, `ref`, `mut`, `&`, parentheses, square brackets, `..`, `const`
 --> tests/fail/expr/let.rs:2:25
  |
2 |     one_assert::assert!(let);
  |                         ^^^

error: incomplete expression: expected `=`
 --> tests/fail/expr/let.rs:3:25
  |
3 |     one_assert::assert!(let b);
  |                         ^^^^^

error: Expected a boolean expression, found a let statement
 --> tests/fail/expr/let.rs:4:25
  |
4 |     one_assert::assert!(let b = true);
  |                         ^^^^^^^^^^^^

error: incomplete expression: unexpected end of input, expected an expression
 --> tests/fail/expr/let.rs:5:25
  |
5 |     one_assert::assert!(let b = 1 +);
  |                         ^^^^^^^^^^^
